<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Polygonal File Viewer</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(30,30,30);
color: white;


</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_viewer">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">padding :0 10 0 0</string>
      </property>
      <property name="text">
       <string>Viewer</string>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_bottomLeft">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(68, 67, 68);</string>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_viewer">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QSplitterHandle:hover {

}  QSplitter::handle:vertical:hover {
background-color: rgb(66, 133, 255);
}</string>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="handleWidth">
       <number>5</number>
      </property>
      <widget class="QTabWidget" name="tabWidget">
       <property name="styleSheet">
        <string notr="true">QTabBar::tab 
{
    background-color: rgb(80, 80, 80);
    color: rgb(133, 133, 133); 
}

QTabBar::tab:selected
{
    color: white;
    background: rgb(30,30,30); 
}
QTabWidget::tab-bar {
    left: 0; 
}</string>
       </property>
       <property name="tabsClosable">
        <bool>true</bool>
       </property>
       <property name="movable">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(231, 195, 44);</string>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar_top">
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolBar {
	border: 0px ;
}
QToolButton {
     border: 0px;
	padding:5px;
}
QToolButton:hover {
	background-color: rgb(255, 255, 255);
}
QToolButton:checked {
	 border: 5px;
}</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::NoToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_selectActor"/>
   <addaction name="action_selectVertex"/>
   <addaction name="action_cutter"/>
   <addaction name="action_clearSelection"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_importMesh"/>
    <addaction name="action_exportMesh"/>
    <addaction name="separator"/>
    <addaction name="action_openExplorerFolder"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuSelection">
     <property name="title">
      <string>Selection</string>
     </property>
     <addaction name="action_clearSelection"/>
    </widget>
    <addaction name="menuSelection"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionExplorer"/>
   </widget>
   <widget class="QMenu" name="menuUMTRI_GUI">
    <property name="title">
     <string>UMTRI GUI</string>
    </property>
    <addaction name="action_preferences"/>
   </widget>
   <addaction name="menuUMTRI_GUI"/>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QToolBar" name="toolBar_home">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolBar {
	border: 0px ;
	background-color: rgb(51, 51, 51);
}
QToolButton {
     border: 0px;
	padding:10px;
	background-color: rgb(51, 51, 51);
}
QToolButton:hover {
	background-color: rgb(255, 255, 255);
}
QToolButton:checked {
	 border: 5px;
}</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::NoToolBarArea</set>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionHome"/>
   <addaction name="actionMonitor"/>
   <addaction name="actionHelp"/>
   <addaction name="action_preferences"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_explorer">
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea|Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Explorer</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_explorer">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_explorer">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>18</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Explorer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_explorer">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>res/folder.png</normaloff>res/folder.png</iconset>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_panelLeft">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTreeView" name="treeView_explorer">
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
       <property name="indentation">
        <number>10</number>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_panelLeft">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>266</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_objects">
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea|Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Objects</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_objects">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_activityMonitor">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Objects</string>
       </property>
       <property name="indent">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_bottomLeft_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(68, 67, 68);</string>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTreeView" name="treeView_projects">
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
       <property name="headerHidden">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_activityMonitor">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>267</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_selectVertex">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>res/pickPoint_off.png</normaloff>
     <normalon>res/pickPoint_on.png</normalon>res/pickPoint_off.png</iconset>
   </property>
   <property name="text">
    <string>selectVertex</string>
   </property>
  </action>
  <action name="action_selectActor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>res/pickActor_off.png</normaloff>
     <normalon>res/pickActor_on.png</normalon>res/pickActor_off.png</iconset>
   </property>
   <property name="text">
    <string>selectActor</string>
   </property>
   <property name="toolTip">
    <string>selectActor</string>
   </property>
  </action>
  <action name="action_importMesh">
   <property name="text">
    <string>Import Mesh...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_exportMesh">
   <property name="text">
    <string>Export Mesh...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="action_clearSelection">
   <property name="icon">
    <iconset>
     <normaloff>res/clearSelection.png</normaloff>res/clearSelection.png</iconset>
   </property>
   <property name="text">
    <string>Clear Selection</string>
   </property>
  </action>
  <action name="actionExplorer">
   <property name="text">
    <string>Explorer</string>
   </property>
  </action>
  <action name="action_preferences">
   <property name="icon">
    <iconset>
     <normaloff>res/settings.png</normaloff>res/settings.png</iconset>
   </property>
   <property name="text">
    <string>Preferences...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+,</string>
   </property>
  </action>
  <action name="action_cutter">
   <property name="icon">
    <iconset>
     <normaloff>res/select_on.png</normaloff>res/select_on.png</iconset>
   </property>
   <property name="text">
    <string>Cutter</string>
   </property>
  </action>
  <action name="action_openExplorerFolder">
   <property name="icon">
    <iconset>
     <normaloff>res/folder.png</normaloff>res/folder.png</iconset>
   </property>
   <property name="text">
    <string>Open Folder...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionHome">
   <property name="icon">
    <iconset>
     <normaloff>res/home.png</normaloff>res/home.png</iconset>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </action>
  <action name="actionMonitor">
   <property name="icon">
    <iconset>
     <normaloff>res/monitor.png</normaloff>res/monitor.png</iconset>
   </property>
   <property name="text">
    <string>Monitor</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset>
     <normaloff>res/help.png</normaloff>res/help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
